modules/ROOT/nav.adoc

```diff
@@ -1,60 +1,99 @@
 .xref:index.adoc[Anypoint Runtime Fabric]
 * xref:index.adoc[Anypoint Runtime Fabric Overview]
+** xref:limitations-self.adoc[Requirements and Limitations]
```

Comment:
General: is there any reason to keep the text inside the brackets? Since you are restructuring the nav, you could just leave the entries auto-populate the title for better maintenance, for example: **xref:limitations-self.adoc[]**. Unless there's a specific reason to keep the names.

Reply:
Yes, because I've shortened the names in the TOC. The headings title in the existing doc are longer (for better SEO).